enterprise/suppress-user/handler_test.go

```diff
@@ -42,7 +45,7 @@ func (*fakeSuppresser) Suppressed(_, _, _ string) (*model.Metadata, error) {
 	if rand.Intn(2)%2 == 0 { // skipcq: GSC-G404
 		return nil, fmt.Errorf("some error")
 	} else {
-		return nil, nil
+		return nil, ErrNoValueFound
```

Comment:
Since it's just a mock, we can return an empty struct? We don't need ErrNoValueFound here in tests?
```suggestion
		return &model.Metadata{}, nil
```

Reply:
We can modify the test to be something like this, since we are just testing the concurrency and don't care about the return value?

```GO
func TestIsSuppressedConcurrency(t *testing.T) {
	log := &tLog{Logger: logger.NOP}
	h := newHandler(&fakeSuppresser{}, log)

	var wg sync.WaitGroup
	wg.Add(1000)
	for i := 0; i < 1000; i++ {
		go func() {
			defer wg.Done()
			h.GetSuppressedUser("workspaceID", "userID", "sourceID")
		}()
	}
	wg.Wait()
	require.Less(t, log.times, 1000)
}
```